src/components/MyIslandPage/IslandMap/IslandMap.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styles from "./IslandMap.module.css";
+
+export const IslandMap = (props: { islandMap: string[][], address: string }) => {
+  const { islandMap, address } = props;
+  return (
+    <div className={styles.container}>
+      <h3>{address}</h3>
+      {islandMap.map((array) => {
+        return (
+          <div>
+            {array.map((element) => {
```

Comment:
find better variable names. to be more descriptive

Reply:
done